apps/hash-external-services/docker-compose.yml

```diff
@@ -117,6 +117,28 @@ services:
     environment:
       COLLECTOR_OTLP_ENABLED: "true"
 
+  type-fetcher:
+    init: true
+    build:
+      dockerfile: docker/Dockerfile
+      context: ../../apps/hash-graph
+      args:
+        PROFILE: dev
+    environment:
+      HASH_GRAPH_LOG_FORMAT: "${HASH_GRAPH_LOG_FORMAT:-pretty}"
+      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-info,hash_type_fetcher=trace,reqwest=debug}"
+      RUST_BACKTRACE: 1
+    entrypoint: "/hash-type-fetcher"
+    command: ["--type-fetcher-host", "0.0.0.0", "--type-fetcher-port", "4444"]
```

Comment:
Any reason for these to be args rather than environment variables? I suppose we're overwriting the existing command, but that can also be empty I assume